WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -1528,8 +1531,8 @@
 /* Title of the view for editing a coupon with percentage discount. */
 "Edit percentage discount" = "Edit percentage discount";
 
-/* Button for specify the product categories where a coupon can be applied in the view for adding or editing a coupon. */
-"Edit Product Categories" = "Edit Product Categories";
+/* Button for specify the product categories where a coupon can be applied in the view for adding or editing a coupon. Reads like: Edit Categories */
+"Edit Product Categories (%1$d)" = "Edit Product Categories (%1$d)";
```

Comment:
Found in the version with the new tooling, too. See [`5c21528`](https://github.com/woocommerce/woocommerce-ios/pull/6737/commits/5c215284d6839f397732f684ec664405f42939ab#diff-96d03942a96e588fcdca0288a2a18d0466a6e44ee5b256f581cd17f2086ea22dL1531-R1616)